addons/account/models/account_move.py

```diff
@@ -3739,6 +3739,18 @@ def name_get(self):
             result.append((line.id, name))
         return result
 
+    @api.model
+    def _name_search(self, name, args=None, operator='ilike', limit=100, name_get_uid=None):
+        if operator == 'ilike':
+            args = ['|', '|',
+                    ('name', '=', name),
```

Comment:
Why not `ilike`? Do you want the exact same name, case sensitive?
```suggestion
                    ('name', 'ilike', name),